db_seed.js

```diff
@@ -30,11 +30,17 @@ const createRandomPhotoUrls = (images) => {
 
   db.sync()
     .then(() => {
-      activeImages.forEach((img, i) => {
-        let key = `url${i}`;
-        photos.create({
-          [key]: img
-        });
+      db.Photos.create({
```

Comment:
Cool. I see that you created one instance of your Photos model. 

I think creating your Photos table like this will not cover a lot of ways the Photos are displayed on the Etsy product page. 

For example all Products won't have the same set of photos. Some product pages might have only 3 photos, some might have 7, etc.... 

I recommend that you create 100 different sets. For example, you could keep the same structure of your Photos table, just make some of the URLS null instead of using your activeImages array. 

So when you do create API calls, the photos that will display will depend on the ID of this photos table.

Reply:
This is inside a function that creates a random number of random images.

Reply:
Woops alrighty cool!